### PR TITLE
added optional snapshot identifer

### PR DESCRIPTION
**Pull request by meyerjrr**
Adds necessary fields needed for making incremental snapshot updates to the Common Fate deployment RDS database.
- snapshot_identifier
- suffix

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -454,3 +454,14 @@ variable "administrator_emails" {
   default     = []
   type        = list(string)
 }
+variable "snapshot_identifier" {
+  description = "(Optional) Specifies whether or not to create this database from a snapshot. This correlates to the snapshot ID you'd find in the RDS console, e.g: rds:production-2015-06-26-06-05."
+  type        = string
+  default     = null
+}
+
+variable "rds-suffix" {
```
Comment (chrnorm) [inline variables.tf:463]:
```suggestion
variable "rds_suffix" {
```

File: variables.tf [hunk 2]
```diff
@@ -454,3 +454,14 @@ variable "administrator_emails" {
   default     = []
   type        = list(string)
 }
+variable "snapshot_identifier" {
```
Comment (chrnorm) [inline variables.tf:457]: This should include 'rds' for clarity

```suggestion
variable "rds_snapshot_identifier" {
```

**Review by chrnorm — DISMISSED** [review]
small nits

File: variables.tf (hunk 1 above)
Comment (meyerjrr) [reply variables.tf:463]: Just double checking here if this was a mistake @chrnorm

**Review by JoshuaWilkes — APPROVED** [review]

**Pull request merged**